src/uiStrings.ts

```diff
@@ -132,5 +132,14 @@ export class UiStrings {
     static readonly GetTreeView = vscode.l10n.t("Please connect to Azure API Center Service first.");
     static readonly APIControlPlaneView = vscode.l10n.t("API Center Management Plane");
     static readonly APIDataPlaneView = vscode.l10n.t("API Center Data Plane");
-    static readonly APIDataPlaneWiki = vscode.l10n.t("API Catalog Guide")
+    static readonly APIDataPlaneWiki = vscode.l10n.t("API Catalog Guide");
+    static readonly GenerateOpenApiProgress = vscode.l10n.t("Invoking Azure API Center to generate OpenAPI specification documentation...");

```

Comment:
Is it Azure API Center or VSCode Azure API Center Extension?

Reply:
Good point. I changed to `Azure API Center plugin`